tools/topology/m4/pipeline.m4

```diff
@@ -33,7 +33,8 @@ define(`W_PIPELINE',
 
 dnl PIPELINE_PCM_ADD(pipeline,
 dnl     pipe id, pcm, max channels, format,
-dnl     frames, deadline, priority, core)
+dnl     frames, deadline, priority, core,
+dnl     schedule_period)
```

Comment:
Ok, this overlaps with "deadline" - and deadline could probably be changed to "period". Your current "schedule_period" could become "initiator"